### DISCUSSION TITLE
Please review my project.

### DISCUSSION THREAD

**Discussion by Constantinno1** [WalletConnect/walletconnect-monorepo#4335] [Developer Support]
I submitted my review 14 days ago and it is still pending, please review my project, I appreciate it.
Thank you!
Project ID: 181a2b132a8c248832a7a9bef6b449a6

**Comment by paritaVhits** [comment]
Hello any one give me idea to register my own wallet in walletconnect ? if any one know then let me know

**Reply by glitch-txs** [reply]
you can follow our docs: https://docs.walletconnect.com/cloud/explorer-submission

**Comment by glitch-txs** [comment]
Hi, dapp submissions will be soon removed, so we are not approving them. This won't affect the WalletConnect integration in any way.

**Reply by Constantinno1** [reply]
So what do I need to do if I want my site to be able to use walletconnect on mobile?

**Reply by glitch-txs** [reply]
You only need to verify your domain and to have a project ID
